src/EIP712/ForwardRequest.ts

```diff
@@ -5,7 +5,7 @@ export interface ForwardRequest {
     from: string;
     to: string;
     tokenContract: string;
-    collectorContract?: string;
+    collectorContract: string;
```

Comment:
are you sure about this? initially i think we set it to optional so that the javascript side could still work without it; eventually setting a default value before reaching the solidity side.

i'm unsure of not having this field optional in terms of backwards code compatibility. especially since we were able to make this work with the dapp without making this a mandatory field